### ISSUE TITLE
Missing Method Exception

### ISSUE THREAD

**Issue by STRML** [BitMEX/ninjatrader-bitmex#11]
Came in from a user:

![image](https://user-images.githubusercontent.com/1197375/35840946-e631a85a-0abe-11e8-9759-48d805454eab.png)

Any ideas?

**Comment by ryanfox** [comment]
The full error message would be helpful, and the full NinjaTrader version number when this came up.

Another thing to double check is that the plugin installed correctly. I have also seen indications there may be issues if you're not running the exact right version of NinjaTrader.

**Comment by matfga** [comment]
I have same problem , I pass you the the version of my ninjatrader8
![image](https://user-images.githubusercontent.com/36211417/35933941-dc882d3a-0c1a-11e8-86ae-e9cb5b06d16b.png)

**Comment by ryanfox** [comment]
How is the error triggered? Can you paste the full error message here?

**Comment by matfga** [comment]
![error bitmex conetc](https://user-images.githubusercontent.com/36211417/36220272-c1b6ce48-1198-11e8-964a-2d054bb1d125.jpg)

**Comment by ryanfox** [comment]
That is the same image as the previous one in this thread. What is the rest of the error message?

What is the application doing when that error comes up?

**Comment by matfga** [comment]
Yes, that user is me, I send that screen cut , that is the error that returns the ninja trader when wanting to establish the connection with Bitmex.
this post was started by a ticket that I made in Bitmex

**Comment by matfga** [comment]
here send the error full ( 2 prints )
![errorninja8 full](https://user-images.githubusercontent.com/36211417/36222967-afff851a-11a1-11e8-8975-56b33165a86d.jpg)

**Comment by ryanfox** [comment]
That's strange, there are no calls to that method in the adapter code. What version of the BitMEX adapter do you have installed?

**Comment by matfga** [comment]
the last   realese 1.0.4

**Comment by ryanfox** [comment]
I haven't been able to reproduce this. Since the ticket was opened, new versions have been released. As of this writing [plugin version 1.0.6](https://github.com/BitMEX/ninjatrader-bitmex/releases/tag/v1.0.6) is the most recent.

Does the issue still occur with the latest plugin version?
